### DISCUSSION TITLE
Accordion component

### DISCUSSION THREAD

**Discussion by bohdan-shulha** [filamentphp/filament#12512] [Ideas]
Hi team,

I would be happy to see the accordion component in Filament standard lib.

My use-case is on the screenshot:

<img width="1246" alt="image" src="https://github.com/filamentphp/filament/assets/533048/28c9597d-8346-4dcb-9195-7710d0ee6fa3">

It is possible to stack collapsible sections, but this is slightly different thing (and looks ugly).

**Comment by leandrocfe** [comment]
https://filamentphp.com/plugins/lara-zeus-accordion
?
